test/testlib/minikube_utilities.go

```diff
@@ -1346,3 +1346,46 @@ func RemoveOrphanNamespaces(t *testing.T) {
 		}
 	}
 }
+
+func DeleteCrashLoopBackOffPods(t *testing.T, namespaceName string, collectLog bool) {
+	toDelete := make(map[string]corev1.ContainerStatus)
+	// find all Pods that have containers reported as CrashLoopBackOff
+	for _, pod := range FindAllPodsInSchema(t, namespaceName) {
+		for _, status := range pod.Status.ContainerStatuses {
+			if status.State.Waiting != nil && status.State.Waiting.Reason == "CrashLoopBackOff" {
+				toDelete[pod.Name] = status
+			}
+		}
+	}
+	for podName, previousStatus := range toDelete {
+		if collectLog {
+			GetAppLog(t, namespaceName, podName, "-crashloopbackoff", &corev1.PodLogOptions{})
```

Comment:
Should this be moved down to right before the `DeletePod()` invocation so that we don't needlessly collect logs?